util/build.json

```
@@ -23,7 +24,8 @@
             "execinfo.h",
             ["file.h", "sys/file.h"],
             ["stdio.h", "stdio.h"],
-            "err.h"
+            "err.h",
+            "unistd.h"
```

```suggestion
            ["wrapper/unistd.h", "unistd.h"],
            "err.h"
```

System header wrappers should go in the wrapper subdirectory now. Also makes it easier to diff if we insert before the last one, since there's no need to add a comma to the previous line.